operator/mina/lib/src/verifier_index.rs

```diff
@@ -17,7 +17,7 @@ use kimchi::{
 };
 use serde::Deserialize;
 
-const BLOCKCHAIN_VK_JSON: &str = include_str!("mainnet_vk.json");
+const BLOCKCHAIN_VK_JSON: &str = include_str!("devnet_vk.json");
```

Comment:
This is because we're running the testnet node currently, right?

Reply:
Yes, but I can revert this change until we parameterize it.

Reply:
its okay !